### DISCUSSION TITLE
[881100] Noita [仅本体]

### DISCUSSION THREAD

**Discussion by Jiransama** [pjy612/SteamManifestCache#111] [Share]
appid:881100
```
881101;c766a099c7245b97c23cf08b9e88e0460813214fde82ce8cc2b80034ec0d1848
```
code:2901C041038E2B8C
email:2854925836@qq.com
[881100.zip](https://github.com/pjy612/SteamManifestCache/files/13748440/881100.zip)

**Comment by pjy612** [comment]
密钥请直接贴出来。
虽然有人合集分享过。
查收

**Reply by Jiransama** [reply]
好的，感谢
